src/pseudopeople/loader.py

```diff
@@ -34,6 +34,10 @@ def load_and_prep_1040_data(data_path: dict, user_filters: List[Tuple]) -> pd.Da
 
     # Load data
     df_1040 = load_standard_dataset_file(data_path[DatasetNames.TAXES_1040], user_filters)
+    # We do not want to filter by state for dependents
```

Comment:
Add why we don't want to filter (i.e. "because dependents might live in a different state from their guardians")